EVENT.md

```diff
@@ -139,8 +139,11 @@ window.open(
 ```
 or via a JavaScript API:
 ```javascript
-window.attributionReporting.registerSource(
-  "https://adtech.example/attribution_source?my_ad_id=123");
+const headers = {
+  'Attribution-Reporting-Eligible': 'true'
+};
+window.fetch("https://adtech.example/attribution_source?my_ad_id=123",
+             { headers });
```

Comment:
Maybe it's just early, but why doesn't this need to be
```
window.fetch("...", {headers: headers})
```
I looked at the MDN docs and they also have this example, but I don't know why it works.

Reply:
This is a shorthand object literal (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#new_notations_in_ecmascript_2015) though we can certainly just inline it for clarity.